### DISCUSSION TITLE
Newcomer

### DISCUSSION THREAD

**Discussion by Beita05** [halkyonio/primaza-poc#75] [General]
Hi everyone, I'm an opensource beginner, I would love to contribute to this project.

**Comment by cmoulliard** [comment]
Why are you interested by this project ? @Beita05

**Reply by zedbeit** [reply]
I just realized that I used my student account to comment but this my main account..

I use quarkus at work, and I'd like to contribute to it, but contributing as a beginner is quite a challenge, so to get familiar with quarkus, I started contributing to a small project that uses quarkus, project [STARFIX](https://github.com/starfixdev/starfix).

Actually I just want to learn and grow while contributing to open source.

**Reply by zedbeit** [reply]
Hi @cmoulliard

**Reply by cmoulliard** [reply]
This project which is also a Quarkus + Qute + HTMX project is a poc (to improve on kubernetes the service binding) and development could end in a couple of weeks. Don't forget that !

**Reply by zedbeit** [reply]
ohh okay.
